### PR TITLE
new survey analysis recipe

### PR DESCRIPTION
**Pull request by danicafine**
Moving to https://github.com/confluentinc/kafka-tutorials/pull/1447 as requested.

### CODE REVIEW

File: _data/harnesses/survey-responses/ksql-test.yml
Comment (davetroiano) [inline _data/harnesses/survey-responses/ksql-test.yml:1]: `table_stream.sql` and `mask.sql` should be run as part of this test

File: _includes/tutorials/survey-responses/ksql-test/code/tutorial-steps/test/expected-outputs/survey_analysis.log
Comment (davetroiano) [inline _includes/tutorials/survey-responses/ksql-test/code/tutorial-steps/test/expected-outputs/survey_analysis.log:6]: >or, you may even be able to `diff` the entire expected output (`survey_analysis.log`) with the actual output.

not sure if the results are in the same reliable order, but if so then a straight `diff` in the Makefile would work for the test condition

File: _includes/tutorials/survey-responses/ksql/markup/dev/ksqlDB.adoc
Comment (davetroiano) [inline _includes/tutorials/survey-responses/ksql/markup/dev/ksqlDB.adoc:20]: change em dash to two dashes

**Review by davetroiano — CHANGES_REQUESTED** [review]
love the recipe! just needs a few fixes mostly around the test

also, could you do this PR as a branch directly off of the `confluentinc` repo rather than from a fork? the automated Semaphore build doesn't get run on PRs from forks

File: _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.json
Comment (bbejeck) [inline _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.json:8]: I could be wrong - but since the source tables and stream specify the incoming value format as `JSON` I think the connector configs need to update `AVRO` -> `JSON`

File: _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.json
Comment (bbejeck) [inline _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.json:23]: ditto

File: _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.sql
Comment (bbejeck) [inline _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.sql:7]: ditto

File: _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.sql
Comment (bbejeck) [inline _includes/tutorials/survey-responses/ksql/code/tutorial-steps/dev/source.sql:21]: ditto

**Review by bbejeck — COMMENTED** [review]
Great tutorial @danicafine - I just have a couple of minor comments this LGTM modulo comments from @davetroiano

**Pull request closed without merging**
